lib/dnsruby/resource/DNSKEY.rb

```diff
@@ -313,6 +313,8 @@ def public_key
           elsif [Algorithms.DSA,
               Algorithms.DSA_NSEC3_SHA1].include?(@algorithm)
             @public_key = dsa_key
+          elsif [Algorithms.ECDSAP256SHA256, Algorithms.ECDSAP384SHA384].include?(@algorithm)
```

Comment:
Might this be worthy of a method in the `Algorithms` class in `code_mappers.rb`, something like this?:

```
def self.ecdsap?(algorithm)
  [ECDSAP256SHA256, ECDSAP384SHA384].include?(algorithm)
end
```

I don't know if that method name is a good one, maybe there's a better one.

Reply:
To me it seemed pretty straightforward choice at first sight as other algorithms were checked using the same pattern. But if you guys think it benefits code readability, I'm happy to do it 🙂